header.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+#include <iostream>
+#include <string>
+#include <vector>
+using namespace std;
+
+
+enum type {
+    TERRESTRIAL,
+    JOVIAN
+};
+class Planet {
+
+public:
+    string name;
+    float mass_in_kg;
+    float orbital_of_velocity;
+    float mean_of_temperature;
+    float lenght_of_day;
+    type planetType;
+    float distance_from_sun;
+
+
+    Planet(string name, float mass_in_kg, float orbital_of_velocity, float mean_of_temperature, float lenght_of_day, float distance_from_sun, type planetType)
```

Comment:
Please change to
`Planet(string name_of_planet, float mass_in_kg, float orbital_of_velocity, float mean_of_temperature, float lenght_of_day, float distance_from_sun, type planetType) `